tests/processes/ProcessingTest.py

```diff
@@ -275,13 +305,7 @@ def test_process_queues(self):
                  Result.from_values('ABear', 'u', 'f', 5, 1),
                  Result.from_values('ABear', 'u', 'f', 6, 1)]},
             {1: [first_global]},
-            {'f': ['first line  # stop ignoring, invalid ignore range\n',
-                   'second line  # ignore all\n',
-                   'third line\n',
-                   "fourth line  # gnore shouldn't trigger without i!\n",
-                   '# Start ignoring ABear, BBear and CBear\n',
-                   '# Stop ignoring\n',
-                   'seventh']},
+            {'f': self.file_dict[self.factory_test_file]},
```

Comment:
please move all of these changes to existing tests to another PR, or at least a separate commit.

when rewriting code, existing tests should not be modified.  that is how you prove , easily , that you correctly matched the old functionality.

these changes make the reviewers life much more difficult as they need to verify that no functional changes were done to the tests.

And even if a reviewer does that, any time anyone else goes and looks at this change, like when chasing down a bug, they need to verify the exact same thing.  Not nice.

Reply:
(note, your separate commit or PR can be before or after this PR)

Reply:
Thanks this was done, and I've confirmed that the commit before the test changes does in fact build : https://travis-ci.org/jayvdb/coala/builds/415063261